README.md

```diff
@@ -90,6 +90,7 @@ go run github.com/LuleaUniversityOfTechnology/2018-project-roaster/cmd/roasterd
 	-dev-mode \
 	-database-source="user=<db user> dbname=<db name> password=<db password> sslmode=disable"
 ```
+Note, if running in windows, you might need additional flags. Such as the `host` flag for the postgres data base, use the same ip as above for windows (run `docker-machine ip` in your docker shell to show it). The redis data base might also need the `-redis-address=sameipasprevious:6379` flag. If you encounter problems use the `-help` flag to see other options.
```

Comment:
> in windows -> on Windows
> data base -> database
> postgres -> PostgreSQL
> ip -> IP
> windows -> Windows
> redis data base -> Redis database
> sameipasprevious -> Uhmm, isn't it Redis specific? Because it's running in a separate container?

Reply:
I agree with all the grammar corrections but i meant but i wrote with the IP. For me on windows, it worked with the same ip as for the PostgreSQL DB. I assume it's because it's run on the same VM in virtualbox.